pkgs/development/python-modules/arrow/1.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder
+, simplejson, typing-extensions, python-dateutil, pytz, pytest-mock, sphinx
+, dateparser, pytestcov, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "arrow";
+  version = "1.0.3";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0793badh4hgbk2c5g70hmbl7n3d4g5d87bcflld0w9rjwy59r71r";
+  };
+
+  propagatedBuildInputs = [ python-dateutil ]
+    ++ lib.optionals (!pythonOlder "3.8") [ typing-extensions ];
+
+  checkInputs = [
+    dateparser
+    pytestCheckHook
+    pytestcov
+    pytest-mock
+    pytz
+    simplejson
+    sphinx
+  ];
+
+  # ParserError: Could not parse timezone expression "America/Nuuk"
```

Comment:
This maybe requires to fix paths to the tzdata file.

Reply:
I mostly copied the "version one" variant from the `default.nix`, which holds an older "version zero" release. By doing so, I also included this exception to the tests.

Now I tried to trace back the error. It seems like within the sandbox, where the test is executed, the `/etc/zoneinfo` symlink is missing; which makes sense. However, `arrow` does not directly queries any tzdata or zoneinfo, but uses the `dateutil` module, which does so. Thus, I would prefer to not touching the internal calling and just keep this one test disabled.